tests/test_main.py

```diff
@@ -278,16 +270,8 @@ def test_movie_service_create_movie_with_genres():
 def test_movie_service_create_movie_with_actors():
     Actor.objects.create(first_name="Keanu", last_name="Reeves")
     Actor.objects.create(first_name="George", last_name="Clooney")
-    create_movie(
-        movie_title="Matrix",
-        movie_description="Matrix description",
-        actors_ids=[2]
-    )
-    create_movie(
-        movie_title="Batman",
-        movie_description="Batman description",
-        actors_ids=[1]
-    )
```

Comment:
No need to do these changes)

Reply:
I don't understand what I did here. I feel like I didn't change anything in the tests :)